pkg/reconciler/kubernetes/tektonpipeline/tektonpipeline.go

```diff
@@ -131,14 +133,21 @@ func (r *Reconciler) ReconcileKind(ctx context.Context, tp *v1alpha1.TektonPipel
 	tp.Status.MarkPreReconcilerComplete()
 
 	// Check if an tekton installer set already exists, if not then create
-	existingInstallerSet := tp.Status.GetTektonInstallerSet()
+	labelSelector := fmt.Sprintf("%s=%s,%s=%s",
```

Comment:
jfyi k8s pkg "k8s.io/apimachinery/pkg/labels"
provides a func to build labels for `ListOptions`
so something like below
```
// created by requirement
	createdByReq, err := labels.NewRequirement("CreatedByKey", selection.Exists, []string{"TektonPipeline"})
	if err != nil {
		panic(err)
	}
	labelSelector := labels.NewSelector().Add(*createdByReq)
// add another version requirement
	labelSelector = labelSelector.Add(versionReq)
	options.LabelSelector = labelSelector.String()
```

Reply:
yup, this is way better. Thanks 🧑‍💻 👍 
the `sprintf` labels are at multiple places. so i shall add an issue so that all of them can be handled together later.

https://github.com/tektoncd/operator/issues/593